src/bundle/Resources/public/js/scripts/core/dropdown.js

```diff
@@ -87,7 +87,9 @@
 
         createSelectedItem(value, label, icon) {
             const container = doc.createElement('div');
-            const selectedItemRendered = this.selectedItemTemplate.replace('{{ value }}', value).replace('{{ label }}', label);
+            const selectedItemRendered = this.selectedItemTemplate
+                .replace('{{ value }}', ibexa.helpers.text.escapeDataset(value))
```

Comment:
I'm not sure about the `escapeDataset` name, because on one hand we are putting this value into the dataset's attribute but on the other, such an escape would be needed for any HTML attribute in a template. Not only for a dataset's attribute. So isn't this just more like `escapeHTMLAttribute`? 🤔